x-pack/test/reporting_api_integration/reporting_and_security/bwc_existing_indexes.ts

```diff
@@ -42,7 +44,7 @@ export default function ({ getService }: FtrProviderContext) {
 
       after('remove index alias', async () => {
         await esArchiver.load('test/functional/fixtures/es_archiver/logstash_functional');
-        await esArchiver.load('test/functional/fixtures/es_archiver/discover');
+        await kibanaServer.importExport.load('test/functional/fixtures/kbn_archiver/discover');
```

Comment:
I think both of these lines should be `unload`?

Also, I don't understand what the call to `cleanupIndexAlias` does.

Reply:
Well, they were originally .load()'s: https://github.com/elastic/kibana/blob/master/x-pack/test/reporting_api_integration/reporting_and_security/bwc_existing_indexes.ts#L44

As for the kbn archiver, yes...that should be an unload.  Great catch.

For `cleanupIndexAlias`, its a Higher Order Function.  When it's called, it adds and index alias, and then it returns a fn, that when called, removes the same index alias...that it created.

It's first applied here: https://github.com/elastic/kibana/blob/0f3f0b1996d3a2f56647b061fa58b44b5cd4c07e/x-pack/test/reporting_api_integration/reporting_and_security/bwc_existing_indexes.ts#L40
> this is where it creates.

Later on, it's called in the after method, cleaning up what it created, that index.

Anyway, so Ill change the kbn archiver call to be an unload.  Same for the logstash call boss?

Reply:
Yes, please unload logstash_functional as well.  

In the parent index file there's 10 tests.  4 of those load logstash_functional.  So another option would be to split those 10 tests into 2 describes and manage the load/unload of logstash_functional in the before/after there for those 4 tests.  That would save loading it 3 times.  But you don't need to do that on this PR unless you want to.

Reply:
Yeah I'd love to that right after this is merged.